packages/python/plotly/plotly/express/_core.py

```diff
@@ -754,6 +754,215 @@ def apply_default_cascade(args):
         args["marginal_x"] = None
 
 
+def _name_heuristic(argument, field_name, used_col_names):
```

Comment:
here we call it `used_col_names` but elsewhere it's `reserved_names`

Reply:
right, there were a few iterations, hence the different names. I will change this.